tests/test_simple.py

```diff
@@ -288,41 +288,41 @@ class RuntimeSharding(unittest.TestCase):
     that the metadata propagates properly and we actually
     download the real key
     """
-    def test_no_shard(self):
-        config = pywren.wrenconfig.default()
-        old_key = config['runtime']['s3_key']
-        prefix, tar_gz = os.path.split(old_key)
-        # Use the staging key to test as it doesn't have shards
-        config['runtime']['s3_key'] = os.path.join("pywren.runtime.staging", tar_gz)
-        wrenexec = pywren.default_executor(config=config)
-
-        def test_func(x):
-            return x + 1
-
-        future = wrenexec.call_async(test_func, 7)
-        result = future.result()
-        base_runtime_key = config['runtime']['s3_key']
-        self.assertEqual(future.run_status['runtime_s3_key_used'],
-                         base_runtime_key)
-
-    def test_shard(self):
-        config = pywren.wrenconfig.default()
-        old_key = config['runtime']['s3_key']
-        prefix, tar_gz = os.path.split(old_key)
-        # Use a runtime that has shards
-        config['runtime']['s3_key'] = os.path.join("pywren.runtime", tar_gz)
-        wrenexec = pywren.default_executor(config=config)
-
-        def test_func(x):
-            return x + 1
-
-        base_runtime_key = config['runtime']['s3_key']
-
-        future = wrenexec.call_async(test_func, 7)
-        result = future.result()
-        # NOTE: There is some probability we will hit the base key ? 
-        self.assertNotEqual(future.run_status['runtime_s3_key_used'], 
-                         base_runtime_key)
+    # def test_no_shard(self):
```

Comment:
Sorry I don't quite understand the question; All future runtimes, including those that are region-specific, will be sharded. Thus eliminating these tests. 

This seems reasonable because: 
1. if you don't shard the runtime you run into performance issues _really_ quickly
2. All "sharding" technically is is sticking the runtime at some locations pointed to by the meta.json. So we could in the future support single-tarball sharding if we wanted with no change in code. In fact, for testing, I often don't fully shard and just have the meta.json point to a single tarball.